bitrise.yml

```diff
@@ -181,10 +181,15 @@ workflows:
         inputs:
         - notify_email_list: $NOTIFY_EMAILS
     - slack@3:
-        is_always_run: false
         inputs:
         - channel: '#tm-mobile-builds'
         - text: iOS Build Succeeded!
+        - webhook_url_on_error: $SLACK_IOS_WEBHOOK
+        - channel_on_error: '#tm-mobile'
```

Comment:
Maybe we will send notifications about fails to #tm-mobile-builds as we do with successful runs?

Reply:
I think the channel is unimportant because of the fact that the caretaker is going to be tagged. However, personally I would prefer if the errors are indeed sent to tm-mobile so that everyone is aware. tm-mobile-builds has so much noise that the error might get lost if you are not the caretaker.